.github/workflows/nightly.yaml

```diff
@@ -100,6 +100,9 @@ jobs:
         GITHUB_ORG: weaveworks-gitops-test
         GITHUB_TOKEN: "${{ secrets.WEAVE_GITOPS_TEST_WEAVEWORKS_WEAVE_GITOPS_BOT_TOKEN }}"
         GITHUB_KEY: "${{ secrets.WEAVE_GITOPS_TEST_WEAVEWORKS_WEAVE_GITOPS_BOT_SSH_KEY }}"
+        GITLAB_ORG: weave-gitops
+        GITLAB_SUBGROUP: weabe-gitops-sub
```

Comment:
`weabe` typo. Could be "correct" if the Gitlab group is also misspelled

Reply:
Actually turns out they both were wrong. Unknown to me weave-gitops is a subgroup